TUTORIAL.md

```diff
@@ -327,44 +337,47 @@ On successful completion of this request, you should receive a JSON packet conta
 ```sh
-$ curl -v -XPUT https://indoor-maps-api.wrld3d.com/v1/edits/EIM-ad578b1f-d3d6-46ed-8945-787527d1efe0?token=dev_auth_token -F comment="my venue comment" -F file="@/path/to/my/file"
+$ curl -v -XPUT https://indoor-maps-api.wrld3d.com/v1/edits/<UUID>?token=<dev_auth_token> -F comment="<my venue comment>" -F file="@</path/to/my/file>"
 ```
-When your submission has been processed we will send you the details required to view it using the WRLD SDK.  
-If at any time you decide you’d rather delete your edit, you can do so with the following command:
+When your submission has been processed we will send you the details required to view it in WRLD.  
+If at any time you decide you’d rather delete your indoor map, you can do so with the following command:
 ```sh
-$ curl -v -XDELETE https://indoor-maps-api.wrld3d.com/v1/edits/EIM-ad578b1f-d3d6-46ed-8945-787527d1efe0?token=dev_auth_token
+$ curl -v -XDELETE https://indoor-maps-api.wrld3d.com/v1/edits/<UUID>?token<=dev_auth_token>
 ```
-If you have any problems, the [cheatsheet](CHEATSHEET.md) might be able to help, or feel free to [raise an issue](https://github.com/wrld3d/indoor-maps-api/issues/new) or get in touch with us at support@wrld3d.com.
+If you have any problems feel free to [raise an issue](https://github.com/wrld3d/indoor-maps-api/issues/new) or get in touch with us at support@wrld3d.com.
 
 ---
 
-Troubleshooting
+  Troubleshooting
 ===================
 #### <a name="multipolygons"/>Multipolygons
-If you are receiving an error related to multipolygons in your submission, it probably refers to one of your units being split into two or more parts. For instance:
+If you are receiving an error related to multipolygons in your submission, first check that you ran the [Multipart to Singleparts](#export-level-to-geojson) conversion step when you exported your layer to geojson.
 
+If the error still occurs, it probably refers to one of your units being split into two or more parts. For instance:
 
 [<img src="/images/tutorial/multipolygon_appearance.png">](/images/tutorial/multipolygon_appearance.png) 
 
 If you are struggling to locate the multipolygon in question, you can use QGIS' **Topology Checker** to find it:
 
-1. Open the Topology Checker panel via *Vector > Topology Checker > Topology Checker*
+1. Enable the Topology Checker plugin with the Plugins > Manage and Install Plugins... dialog.
+2. Open the Topology Checker panel via *Vector > Topology Checker* 
 
   [<img src="/images/tutorial/topology_checker_location.png">](/images/tutorial/topology_checker_location.png)
-2. Click the *Configure* button (Wrench icon)
-3. Under *Current Rules* set the layer you want to check and set the rule to *"must not have multi-part geometries"*
-4. Click *Add Rule* then click *OK*
+3. Click the *Configure* button (Wrench icon)
```

Comment:
The ordered list item 3 appears just on the left on the image, I'm not sure if it will break the markdown, but can we have a new line after the image?